### ISSUE TITLE
Accessibility not passing up to each CCInput

### ISSUE THREAD

**Issue by jasoneo016** [sbycrosz/react-native-credit-card-input#79]
I am trying to automate some tests on Appium and I am able to create accessibilityLabel, accessibilityTraits, and accessible up to other Components, but when I try to do it using your open source library, it doesn't work. Have you guys experienced this issue before and how would you go about resolving it?

Thanks.

**Comment by fr1n63** [comment]
I think no additionalInputProps are going in to the CCInput TextField - I've been trying to pass up `editable=false` with no joy. It makes it to the CCInput fine, but not on to the TextInput.

Looking at the src of CCInput I can't see any obvious reason why that wouldn't be passed.

```
const inputDisabledProp = {editable: false};

<CreditCardInput
  additionalInputsProps={{
    number: inputDisabledProp,
    expiry: inputDisabledProp,
    cvc: inputDisabledProp
  }}
/>
```

**Comment by triage** [comment]
any update on this one? I'm also stuck with UI tests. The fields aren't visible in `XCUIApplication().textFields`.
They are found in `po XCUIApplication().otherElements` but I can't type into them.

**Comment by triage** [comment]
```html
      // <TouchableOpacity onPress={this.focus}> <-- this line is causing the problem
        <View style={[containerStyle]}>
          { !!label && <Text style={[labelStyle]}>{label}</Text>}
          <TextInput ref="input"
```
making a PR
